<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operate_score</class>
 <widget class="QTabWidget" name="operate_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师管理系统</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>软件工程</string>
   </attribute>
   <widget class="QPushButton" name="Btn_add_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_erase_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ave_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平均分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_pass_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>及格率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_BZC_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>标准差</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>教师</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>人工智能</string>
   </attribute>
   <widget class="QPushButton" name="Btn_erase_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ave_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平均分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_pass_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>及格率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_add_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_BZC_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>标准差</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>教师</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>none</string>
   </attribute>
   <widget class="QPushButton" name="Btn_erase_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ave_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平均分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_pass_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>及格率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_add_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_BZC_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>标准差</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>教师</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_4">
   <attribute name="title">
    <string>none</string>
   </attribute>
   <widget class="QPushButton" name="Btn_ave_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平均分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_add_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>教师</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_pass_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>及格率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_erase_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_BZC_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>标准差</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
